components/ds/tabs/tabs.tsx

```diff
@@ -62,24 +73,27 @@ export function Tabs<T extends Key>({ tabs, color, border, mobile, controlled }:
   }
 
   return (
-    <NextTabs
-      aria-label="Options"
-      variant="underlined"
-      selectedKey={controlled?.selected || selected}
-      onSelectionChange={onSelectTab}
-      classNames={{
-        base: "w-full",
-        tabList: cn("gap-8 w-full relative rounded-none p-0 px-4", slots.tabList()),
-        cursor: "w-full bg-underline h-1",
-        tab: "relative max-w-fit px-0 h-auto pt-[2px] pb-2.5 data-[hover=true]:opacity-100",
-        tabContent: "",
-      }}
-    >
-      {tabs.map(t => (
-        <NextTab {...t} key={t.key} title={<Tab color={color} {...t} />}>
-          {t.children?.(selected as T)}
-        </NextTab>
-      ))}
-    </NextTabs>
+    <>
+      <NextTabs
+        aria-label="Options"
+        variant="underlined"
+        selectedKey={controlled?.selected || selected}
+        onSelectionChange={onSelectTab}
+        classNames={{
+          base: "w-full",
+          tabList: cn("gap-8 w-full relative rounded-none p-0 px-4", slots.tabList()),
+          cursor: "w-full bg-underline h-1",
+          tab: "relative max-w-fit px-0 h-auto pt-[2px] pb-2.5 data-[hover=true]:opacity-100",
+          tabContent: "",
+        }}
+      >
+        {/* eslint-disable-next-line @typescript-eslint/no-unused-vars */}
+        {tabs.map(({ children, ...t }) => (
```

Comment:
To avoid disabling eslint you can rename the variable with an underscore.
```suggestion
        {tabs.map(({ children: _c, ...t }) => (
```